Tests/Rules/AvoidUnInitializedVarsInNewRunspaces.tests.ps1

```diff
@@ -0,0 +1,90 @@
+$directory = Split-Path -Parent $MyInvocation.MyCommand.Path
+$testRootDirectory = Split-Path -Parent $directory
+
+Import-Module (Join-Path $testRootDirectory "PSScriptAnalyzerTestHelper.psm1")
+
+$ruleName = "PSAvoidUnInitializedVarsInNewRunspaces"
+
+$settings = @{
+    IncludeRules = @($ruleName)
+}
+
+Describe "AvoidUnInitializedVarsInNewRunspaces" {

```

Comment:
Generally: Good test coverage and clean code. That's the right level we look for. At the end we might come back and add a few special test cases but looks great so far.